install/updates/preconfig/0.9/preconfig_0.9.inc.php

```diff
@@ -708,4 +708,12 @@ function parseAndOutputPreconfig(&$has_preconfig, &$return, $current_version, $c
 	    $question.= makeyesno('enable_letsencrypt', '1', '0', '1').'<br />';
 	    eval("\$return.=\"" . getTemplate("update/preconfigitem") . "\";");
 	}
+	
+	if(versionInUpdate($current_db_version, '201603150')) {
+	    $has_preconfig = true;
+	    $description  = 'You can chose which TLS versions will be enabled for all your webservers SSL vhosts.<br /><br />';
+	    $question = '<strong>Please select the desired TLS versions below (default: TLSv1,TLSv1.1,TLSv1.2):</strong>&nbsp;';
+	    $question .= '<input type="text" id="update_ssl_protocols" value="TLSv1,TLSv1.1,TLSv1.2"/>'
```

Comment:
why make this a text-field now when the setting is a multiple-select-box? People can enter values as they like which will not be available for selection in the settings -> no good